modules/ROOT/pages/index.adoc

```diff
@@ -21,6 +25,7 @@ Anypoint Studio uses your system's native browser to display elements of the Any
 ** Other supported runtime versions work fine with both Java 7 or 8.
 
 * Studio 6 is not compatible with Mule 4.
+* From Studio 6 on, in Windows, you cannot use Exchange UI or deploy Mule apps to CloudHub.
```

Comment:
also, it really isn't from Studio 6 that it is not supported... it is since IE is not supported anymore on the Platform as previously commented... this doesn't mean that from one day to another all the platform doesn't work... for example there are parts that still work but some services like Exchange doesn't allow anymore to access with IE.

Reply:
Good catch on your first comment!
Regarding IE, tough some parts may work, if we are stating that IE is not compatible with our platform, we are not responsible for the outcome of using our platform with IE, i.e.: it may work, it may not work, or it may render an unexpected result.
Thanks!